### ISSUE TITLE
People page: Accessibility issue on filter button

### ISSUE THREAD

**Issue by Jihed525** [Meeds-io/meeds#2840]
Plf:7.0
Current: 
The filter button does not have an aria label

![Image](https://github.com/user-attachments/assets/089f5b7f-1fe3-4f9a-9f13-584dacb98e94)

Expected:
Apply the information content present in the tool-tip button to the label aria

eXo(maint):71145
